src/components/OnThisPageNav.vue

```diff
@@ -0,0 +1,160 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2021-2022 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+<template>
+  <div class="OnThisPageNav">
+    <slot
```

Comment:
What's the slot used for here?

Reply:
Interesting. Got it, thanks

Reply:
I also refactored this out as well. We can add this later if someone needs it.